client/signup/config/flows-pure.js

```diff
@@ -76,7 +76,7 @@ export function generateFlows( {
 		},
 		{
 			name: 'account',
-			steps: [ userSocialStep, 'set-reader-landing' ],
+			steps: [ userSocialStep ],
```

Comment:
Hey @escapemanuele, just happened to see this PR. I was wondering why `'set-reader-landing'` was removed as a part of this change?

Reply:
Oh my, I must have slipped the change by mistake, I'll add it back now

Reply:
https://github.com/Automattic/wp-calypso/pull/100770